### PR TITLE
fix grafana config and google auth

### PR DESCRIPTION
**Pull request by akastav**

### CODE REVIEW

**Review by RustamGimadiev — CHANGES_REQUESTED** [review]
please format your code

File: modules/monitoring/prometheus/main.tf [hunk 1]
```diff
@@ -141,22 +141,28 @@ locals {
...
-    "rbac.create"                = true,
-    "resources.limits.cpu"       = "100m",
-    "resources.limits.memory"    = "300Mi",
-    "resources.requests.cpu"     = "100m",
-    "resources.requests.memory"  = "300Mi"
+    "alertmanager.enabled"                                        = true
+    "grafana.enabled"                                             = true
+    "grafana.pvc.enabled"                                         = true
+    "grafana.ingress.enabled"                                     = true
+    "grafana.ingress.hosts[0]"                                    = "grafana.${var.domains[0]}"
```
Comment (RustamGimadiev) [inline modules/monitoring/prometheus/main.tf:148]: what if we want to pass empty `var.domains` and access grafana via port-forwarding? seems like we will face index error here, what do you think?

File: modules/monitoring/prometheus/main.tf [hunk 2]
```diff
@@ -34,7 +39,7 @@ resource "kubernetes_secret" "grafana_auth" {
     var.module_depends_on
   ]
 
-  count = var.grafana_google_auth == true ? 1 - local.argocd_enabled : 0
+  count = var.grafana_client_id > 0 ? 1 : 0
```
[the remark this answers is not in the document]
Comment (akastav) [reply modules/monitoring/prometheus/main.tf:42]: Fixed it. We check for the presence of the client id, but not to complicate it, we do not check for the presence of secret. In general, it should not be empty

File: modules/monitoring/prometheus/main.tf (hunk 1 above)
Comment (akastav) [reply modules/monitoring/prometheus/main.tf:148]: we have a local domain by default, it's not empty

**Comment by gfrntz** [comment]
```
terraform version
Terraform v0.14.4
+ provider registry.terraform.io/hashicorp/aws v3.20.0
+ provider registry.terraform.io/hashicorp/external v1.2.0
+ provider registry.terraform.io/hashicorp/helm v1.3.2
+ provider registry.terraform.io/hashicorp/kubernetes v1.13.3
+ provider registry.terraform.io/hashicorp/local v1.4.0
+ provider registry.terraform.io/hashicorp/null v2.1.2
+ provider registry.terraform.io/hashicorp/random v2.3.1
+ provider registry.terraform.io/hashicorp/template v2.2.0
```

DEBUG log
```
k.module.vpc (expand)
module.network.module.vpc.var.enable_ipv6 (expand)
  module.network.module.vpc (expand)
module.network.module.vpc.var.enable_kinesis_firehose_endpoint (expand)
  module.network.module.vpc (expand)
module.network.module.vpc.var.enable_kinesis_streams_endpoint (expand)
  module.network.module.vpc (expand)
module.network.module.vpc.var.enable_kms_endpoint (expand)
  module.network.module.vpc (expand)
module.network.module.vpc.var.enable_lambda_endpoint (expand)
  module.network.module.vpc (expand)
module.network.module.vpc.var.enable_logs_endpoint (expand)
  module.network.module.vpc (expand)
module.network.module.vpc.var.enable_monitoring_endpoint (expand)
  module.network.module.vpc (expand)
module.network.module.vpc.var.enable_nat_gateway (expand)
  module.network.module.vpc (expand)
module.network.module.vpc.var.enable_public_redshift (expand)
  module.network.module.vpc (expand)
module.network.module.vpc.var.enable_public_s3_endpoint (expand)
  module.network.module.vpc (expand)
module.network.module.vpc.var.enable_qldb_session_endpoint (expand)
  module.network.module.vpc (expand)
module.network.module.vpc.var.enable_rds_endpoint (expand)
  module.network.module.vpc (expand)
module.network.module.vpc.var.enable_rekognition_endpoint (expand)
  module.network.module.vpc (expand)
module.network.module.vpc.var.enable_s3_endpoint (expand)
  module.network.module.vpc (expand)
module.network.module.vpc.var.enable_sagemaker_api_endpoint (expand)
  module.network.module.vpc (expand)
module.network.module.vpc.var.enable_sagemaker_notebook_endpoint (expand)
  module.network.module.vpc (expand)
module.network.module.vpc.var.enable_sagemaker_runtime_endpoint (expand)
  module.network.module.vpc (expand)
module.network.module.vpc.var.enable_secretsmanager_endpoint (expand)
  module.network.module.vpc (expand)
module.network.module.vpc.var.enable_servicecatalog_endpoint (expand)
  module.network.module.vpc (expand)
module.network.module.vpc.var.enable_ses_endpoint (expand)
Error: Cycle: module.monitoring.kubernetes_namespace.this, module.monitoring.local.namespace (expand)
xpand)
```

**Pull request closed without merging**